CONTRIBUTING.md

```diff
@@ -18,9 +18,12 @@ licensing restrictions.
 
 ## Code of Conduct
 
-Before contributing please read the [code of conduct](./CODE_OF_CONDUCT.md)
+Before contributing please read the [code of
+conduct](https://github.com/newrelic/newrelic-repo-template/blob/main/CODE_OF_CONDUCT.md)
 
-Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms
+Note that our [code of
+conduct](https://github.com/newrelic/newrelic-repo-template/blob/main/CODE_OF_CONDUCT.md)
```

Comment:
Code of conduct link should be: https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md

Reply:
I'm trying to find where I even got that link so I could fix the source, and I can't even find it. I thought I got it from the oss-template or another one of our repos..🤷 oh well, fix coming soon, thanks for catching that

Reply:
pushed